### ISSUE TITLE
Rules for non-existing namespaces (at the first time) will be ignored forever

### ISSUE THREAD

**Issue by stefanandres** [FairwindsOps/rbac-manager#106]
Scenario:
* a rbacdefinition is created
  * the referenced namespace does not exist
* rbac-manager fails to create rolebinding
  * this is expected
* the namespace is created
  * rbac-manager is not trying to create the rolebinding

```
time="2019-11-12T15:08:38Z" level=info msg="Creating Role Binding: network-access-network-access-edit"
time="2019-11-12T15:08:38Z" level=error msg="Error creating Role Binding: namespaces \"mynamespace\" not found"
```

I'd expect rbac-manager to keep trying and recreate the RoleBinding when the namespace has been created.

**Comment by sudermanjr** [comment]
@stefanandres  Thank you for reporting this. We will take a look at some point.  

I believe that we should be able to fix this issue and https://github.com/FairwindsOps/rbac-manager/issues/53 at the same time.

**Comment by lucasreed** [comment]
This should be fixed now with #128
